### PR TITLE
Add interactive element to phenotypes page

### LINKED ISSUE #55: pathology data- interactive plots?
Can we add an interactive plot feature to the pathology data plots? Similar to the volcano plots.

### LINKED ISSUE #70: incorporate new version of pathology data file
One of the pathology data files has been updated to exclude one subject: syn22049825

### PR DESCRIPTION
**Pull request by sharlagelfand**
This PR closes #55 by adding an interactive element to the phenotypes plots - when you click on a panel, a modal pops up with an interactive version of that panel that can be zoomed, hovered to show points / box plots attributes, etc.

It also simplifies the `magora_boxplot()` code to remove all the logic that allowed that function to be used for both phenotypes and gene expression data, since we aren't using it for gene expressions anymore (using volcano plots and heatmaps instead).

Also address #46 (the pathology page portion only) by updating the metadata, #44 (for pathology only), #58 (uses "mouse model" for the pathology page) and closes #70.

There are a few issues (described in #46 and #44) with the new metadata, but @annagree will address those and once there is another new copy of the metadata, I'll update here - but figured I'd get the PR started since I don't expect the actual code / features to change.

### CODE REVIEW

**Comment by mattfazza** [comment]
I'll review this as soon as I come back from the dentist tomorrow morning 👀

File: data-raw/phenotypes.R [hunk 1]
```diff
@@ -56,7 +59,12 @@ phenotype_data <- phenotype_files %>%
 
 ## Biospecimen metadata ----
 
-# synGet("syn18876530", version = 6, downloadLocation = here::here("data-raw", "pathology"))
+biospecimen_id <- "syn18876530"
```
Comment (mattfazza) [inline data-raw/phenotypes.R:62]: I'm assuming these hard-coded locations don't change often?

**Review by mattfazza — APPROVED** [review]
Couldn't catch anything, Sharla!  Looks good to me! 👀 💯

File: data-raw/phenotypes.R (hunk 1 above)
Comment (sharlagelfand) [reply data-raw/phenotypes.R:62]: No I don't think so! I would be surprised if they changed

**Comment by sharlagelfand** [comment]
Thank you @mattfazza!

**Pull request merged**